Sources/StreamChat/Config/ChatClientConfig.swift

```diff
@@ -186,9 +186,17 @@ public struct ChatClientConfig {
     /// whenever a new channel is created,/updated the SDK will try to
     /// match the channel list filter automatically.
     public var isChannelAutomaticFilteringEnabled: Bool = true
-
-    public init(apiKey: APIKey) {
+    
+    /// The `URLSessionConfiguration` being used as default configuration for the `APIClient` and
+    /// `WebSocketClient`
+    public let urlSessionConfiguration: URLSessionConfiguration
+
+    public init(
+        apiKey: APIKey,
+        urlSessionConfiguration: URLSessionConfiguration = .default
+    ) {
```

Comment:
Just for consistency purposes it is better we follow the rest of the properties approach, where they are mutable with an initial value, and then you can change it whenever you create the config. 👍 

```suggestion
    /// The `URLSessionConfiguration` being used as the default configuration for the `APIClient` and
    /// `WebSocketClient`
    public var urlSessionConfiguration: URLSessionConfiguration = .default

    public init(
        apiKey: APIKey
    ) {
```

Reply:
Consistency is king 🤴 - fixed